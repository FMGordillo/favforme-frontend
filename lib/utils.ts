/**
 * @default 0
 */
export const toPascalCase = (string: string | undefined): string =>
  string
    ? string.replace(/(\w)(\w*)/g, function (_g0, g1, g2) {
        return g1.toUpperCase() + g2.toLowerCase();
      })
    : "";
