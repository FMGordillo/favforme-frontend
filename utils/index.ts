// export const isNotProd = process.env.NEXT_PUBLIC_ENVIRONMENT !== "production";
export const isNotProd = true;
