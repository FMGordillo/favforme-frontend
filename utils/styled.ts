import { DefaultTheme } from "styled-components";

export type Direction = "left" | "center" | "right";

// TODO: Use this from styled.ts
export type Color = "primary" | "secondary" | "gray" | "white" | "black";
export type Variant = "light" | "main" | "dark";
export type Spacing = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
export type Breakpoint = "xs" | "sm" | "md" | "lg" | "xl";

export const calculateBreakpoint = (key: Breakpoint): string => {
  switch (key) {
    case "xs":
      return "0px";
    case "sm":
      return "600px";
    case "md":
      return "960px";
    case "lg":
      return "1280px";
    case "xl":
      return "1920px";
  }
};

const defaultTheme = {
  spacing: (digit: Spacing): string => `${digit * 4}`,
  breakpoints: {
    down: (key: Breakpoint): string =>
      `@media (max-width: ${calculateBreakpoint(key)})`,
    // @deprecated
    up: (key: Breakpoint): string =>
      `@media (min-width: ${calculateBreakpoint(key)})`,
  },
};

export const lightTheme: DefaultTheme = {
  mode: "light",
  palette: {
    primary: { main: "#4f6be6", light: "#9BAAF6", dark: "#27358d" },
    secondary: {
      light: "#FFBCC4",
      main: "#f38d9f",
      dark: "#a21d44",
    },
    gray: { light: "#f6f6f6", main: "#B2B2B2", dark: "#797979" },
    black: {
      light: "#000",
      main: "#000",
      dark: "#000",
    },
    white: {
      light: "#FFF",
      main: "#FFF",
      dark: "#FFF",
    },
    error: "#b50404",
  },
  ...defaultTheme,
};

export const darkTheme: DefaultTheme = {
  mode: "dark",
  palette: {
    // primary: { main: "#5573f5", light: "#9BAAF6", dark: "#27358d" },
    primary: { main: "#27358d", light: "#27358d", dark: "#27358d" },
    secondary: {
      light: "#FFBCC4",
      main: "#f38d9f",
      dark: "#a21d44",
    },
    // gray: { light: "#f6f6f6", main: "#B2B2B2", dark: "#9b9b9b" },
    gray: { light: "#9b9b9b", main: "#9b9b9b", dark: "#2f2f2f" },
    black: {
      light: "#000",
      main: "#000",
      dark: "#000",
    },
    white: {
      light: "#000",
      main: "#000",
      dark: "#000",
    },
    error: "#b50404",
  },
  ...defaultTheme,
};
