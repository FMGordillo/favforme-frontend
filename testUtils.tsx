import { render } from "@testing-library/react";
import { ThemeProvider } from "styled-components";
import { theme } from "./utils/styled";
// import { ThemeProvider } from "my-ui-lib"
// import { TranslationProvider } from "my-i18n-lib"
// import defaultStrings from "i18n/en-x-default"

const Providers = ({ children }) => {
  // return children;
  return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
};

const customRender = (ui, options = {}) =>
  render(ui, { wrapper: Providers, ...options });

// re-export everything
export * from "@testing-library/react";

// override render method
export { customRender as render };
