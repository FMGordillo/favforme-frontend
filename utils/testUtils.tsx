import { render } from "@testing-library/react";
import { ReactChild } from "react";
import { ThemeProvider } from "styled-components";
import { SWRConfig } from "swr";
import { fetcher } from "../lib/queries";
import { lightTheme } from "./styled";

const Providers = ({ children }: { children: ReactChild }) => {
  return (
    <SWRConfig
      value={{
        refreshInterval: 10000,
        fetcher,
      }}
    >
      <ThemeProvider theme={lightTheme}>{children}</ThemeProvider>
    </SWRConfig>
  );
};

// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
const customRender = (ui: JSX.Element, options = {}) =>
  // @ts-ignore
  render(ui, { wrapper: Providers, ...options });

import "@testing-library/jest-dom";
export * from "@testing-library/react";
export { customRender as render };
