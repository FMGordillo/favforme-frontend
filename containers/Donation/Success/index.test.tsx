import { DonationSuccessContainer } from "./";
import { render } from "@/utils/testUtils";

describe("[Container] Donation success", () => {
  it("renders correctly", async () => {
    const { asFragment } = render(
      <DonationSuccessContainer loading={false} donation={null} />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
