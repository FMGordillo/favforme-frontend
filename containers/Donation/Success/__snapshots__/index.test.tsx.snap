// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Container] Donation success renders correctly 1`] = `
<DocumentFragment>
  .c13 {
  border: none;
  cursor: pointer;
  font-size: 1.5rem;
  padding: 12px 28px;
  border-radius: 64px;
  font-family: abel,sans-serif;
  -webkit-transition: background-color 300ms ease-out,color 300ms ease-out;
  transition: background-color 300ms ease-out,color 300ms ease-out;
  background-color: #4f6be6;
}

.c13 > a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c13:disabled {
  cursor: not-allowed;
  background-color: #9BAAF6;
}

.c12 {
  font-size: 2em;
  font-weight: 400;
  color: #4f6be6;
}

.c10 {
  margin: 4em 0;
  margin: 0 4em;
}

.c14 {
  background-color: #4f6be6;
}

.c19 {
  display: grid;
  max-width: 400px;
  padding: 1em;
  margin: 0 auto;
  text-align: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  justify-items: center;
  grid-template-columns: 1fr 1fr 1fr 1fr;
}

.c19 > * {
  color: white;
  -webkit-transition: all 300ms;
  transition: all 300ms;
}

.c19 > *:hover {
  cursor: pointer;
}

.c19 > * > * {
  width: 2em !important;
  height: 2em !important;
}

.c19 > *:hover {
  color: #27358d;
}

.c15 {
  display: grid;
  grid-template-columns: repeat(2,1fr);
  grid-gap: 1em;
  margin: 0 auto;
  padding: 2em 4em;
  max-width: 1024px;
}

.c15 > :first-child {
  justify-self: center;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

.c16 {
  display: grid;
  grid-template-columns: repeat(2,1fr);
  list-style: none;
  padding: 0;
}

.c17 {
  line-height: 2.5;
}

.c18 {
  color: white;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: uppercase;
  border-bottom: 1px solid transparent;
  -webkit-transition: all ease-in-out 300ms;
  transition: all ease-in-out 300ms;
}

.c18:hover {
  border-color: white;
}

.c20 {
  background-color: #27358d;
  text-align: center;
}

.c20 p {
  margin: 0;
  padding: 28px 0 16px;
  color: white;
  text-transform: uppercase;
}

.c20 .credits {
  font-size: 14px;
  text-transform: none;
  padding: 0 0 24px;
  color: #4f6be6;
}

.c20 a {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #9BAAF6;
}

.c0 {
  background-color: #FFF;
}

.c7 {
  background-size: cover;
  background-position: center;
  background-image: url("/images/plato_de_comida.webp");
  height: 400px;
  width: 100vw;
  overflow: hidden;
  display: table-cell;
  vertical-align: bottom;
  padding-bottom: 48px;
  -webkit-transition: height 300ms;
  transition: height 300ms;
}

.c8 {
  color: white;
  -webkit-transform: translateY(-64px);
  -ms-transform: translateY(-64px);
  transform: translateY(-64px);
  padding: 0;
  -webkit-transition: -webkit-transform 300ms;
  -webkit-transition: transform 300ms;
  transition: transform 300ms;
}

.c9 {
  margin: 0;
  font-size: 50px;
  text-overflow: ellipsis;
  overflow: hidden;
  width: inherit;
  text-align: center;
}

.c9 mark {
  color: white;
  background-color: #4f6be6;
}

.c1 {
  position: absolute;
  left: 0;
  right: 0;
  z-index: 9;
  max-height: 250px;
  border-radius: 0 0 100% 100%/0 0 25% 25%;
  -webkit-transition: all 300ms;
  transition: all 300ms;
}

.c2 {
  display: none;
  position: fixed;
  top: 0;
  z-index: 9;
  height: 4em;
  width: 100vw;
  padding: 1em;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  list-style: none;
  padding: 0;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c4 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  font-family: dosis,sans-serif;
  text-transform: uppercase;
  font-size: 15px;
  font-weight: 500;
  text-align: center;
  max-width: 200px;
  display: block;
}

.c4 > a > div {
  max-width: 150px;
}

.c4 a {
  color: white;
}

.c6 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  font-family: dosis,sans-serif;
  text-transform: uppercase;
  font-size: 15px;
  font-weight: 500;
  text-align: center;
  max-width: 200px;
  display: block;
}

.c6 > a > div {
  max-width: 150px;
}

.c6 a {
  color: #f38d9f;
}

.c5 {
  font-weight: 500;
  color: white;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  border-bottom: 1px solid transparent;
  -webkit-transition: all 300ms;
  transition: all 300ms;
}

.c5:hover {
  font-weight: 700;
  color: #f38d9f;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 32px;
}

.c11 p {
  text-align: center;
}

@media (max-width:600px) {
  .c10 {
    margin: 0 2em;
  }
}

@media (max-width:600px) {
  .c15 {
    grid-template-columns: 1fr;
  }

  .c15 > :last-child {
    text-align: center;
    justify-self: center;
  }
}

@media (max-width:960px) {
  .c16 {
    grid-template-columns: 1fr;
  }
}

@media (max-width:960px) {
  .c7 {
    height: 300px;
  }
}

@media (max-width:960px) {
  .c8 {
    -webkit-transform: translateY(0px);
    -ms-transform: translateY(0px);
    transform: translateY(0px);
  }
}

@media (max-width:600px) {
  .c8 {
    padding: 0;
  }
}

@media (max-width:960px) {
  .c9 {
    width: inherit;
  }
}

@media (max-width:600px) {
  .c1 {
    color: red;
  }
}

@media (max-width:600px) {
  .c2 {
    display: block;
  }
}

@media (max-width:600px) {
  .c3 {
    -webkit-transform: translateY(-100%);
    -ms-transform: translateY(-100%);
    transform: translateY(-100%);
    background: #27358d;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    padding-top: 80px;
    margin: 0;
    -webkit-transition: -webkit-transform 250ms ease-in;
    -webkit-transition: transform 250ms ease-in;
    transition: transform 250ms ease-in;
    position: fixed;
    width: 100vw;
  }

  .c3 > :nth-child(3) {
    display: none;
  }

  .c3 > :nth-child(3) {
    display: none;
  }

  .c3 > * {
    margin: 1em;
  }

  .c3 > :last-child {
    margin-bottom: 2em;
  }
}

@media (max-width:600px) {
  .c4 {
    display: block;
  }
}

@media (max-width:600px) {
  .c6 {
    display: block;
  }
}

<main
    class="c0"
  >
    <nav
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          style="overflow: hidden; box-sizing: border-box; display: inline-block; position: relative; width: 60px; height: 60px;"
        >
          <img
            data-nimg="fixed"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </div>
      </div>
      <ul
        class="c3"
      >
        <li
          class="c4"
        >
          <a
            class="c5"
          >
            FavForMe
          </a>
        </li>
        <li
          class="c4"
        >
          <a
            class="c5"
          >
            Acciones
          </a>
        </li>
        <li
          class="c4"
        >
          <a
            class="c5"
          >
            <div
              style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
            >
              <div
                style="box-sizing: border-box; display: block; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjcwIiBoZWlnaHQ9IjIxNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                  style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                />
              </div>
              <img
                alt="FavForMe logo"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </div>
          </a>
        </li>
        <li
          class="c4"
        >
          <a
            class="c5"
          >
            Empresas
          </a>
        </li>
        <li
          class="c6"
        >
          <a
            class="c5"
          >
            Contacto
          </a>
        </li>
      </ul>
    </nav>
    <header
      class="c7"
    >
      <div
        class="c8"
      >
        <h1
          class="c9"
        >
          Donación exitosa
        </h1>
      </div>
    </header>
    <section
      class="c10 c11"
    >
      <h1
        class="c12"
      >
        ¡Felicitaciones!
      </h1>
      <p>
        Se acreditó tu donación de 
        <strong>
          $.-
        </strong>
        <br />
        En la acción 
        <strong />
      </p>
      <div
        style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
      >
        <div
          style="box-sizing: border-box; display: block; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzUwIiBoZWlnaHQ9IjQwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
            style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
          />
        </div>
        <img
          alt="Personas celebrando"
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </div>
      <p>
        Gracias a tu ayuda estamos un paso más cerca de alcanzar la
        <br />
        meta para que 
        <strong>
           pueda cumplir su objetivo.
        </strong>
      </p>
      <h1
        class="c12"
      >
        ¡Muchas gracias!
      </h1>
      <button
        class="c13"
      >
        Volver
      </button>
    </section>
    <section
      class="c14"
    >
      <div
        class="c15"
      >
        <div>
          <a
            aria-label="Volver al inicio"
            href="#"
          >
            <div
              style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
            >
              <div
                style="box-sizing: border-box; display: block; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAwIiBoZWlnaHQ9IjU1IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                  style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                />
              </div>
              <img
                alt="Logo FavForMe"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </div>
          </a>
        </div>
        <ul
          class="c16"
        >
          <li
            class="c17"
          >
            <a
              class="c18"
              href="/acciones"
            >
              Acciones
            </a>
          </li>
          <li
            class="c17"
          >
            <a
              class="c18"
              href="/nosotros"
            >
              FavForMe
            </a>
          </li>
          <li
            class="c17"
          >
            <a
              class="c18"
              href="/#brands"
            >
              Empresas
            </a>
          </li>
          <li
            class="c17"
          >
            <a
              class="c18"
            >
              Contacto
            </a>
          </li>
          <li
            class="c17"
          >
            <a
              class="c18"
              href="/terms_and_conditions"
            >
              Términos de uso
            </a>
          </li>
        </ul>
      </div>
      <div
        class="c19"
      >
        <a
          aria-label="LinkedIn"
          href="https://www.linkedin.com/company/64580065"
          rel="noreferrer noopener"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-linkedin-in fa-w-14 "
            data-icon="linkedin-in"
            data-prefix="fab"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
              fill="currentColor"
            />
          </svg>
        </a>
        <a
          aria-label="Instagram"
          href="https://www.instagram.com/favforme"
          rel="noreferrer noopener"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-instagram fa-w-14 "
            data-icon="instagram"
            data-prefix="fab"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
              fill="currentColor"
            />
          </svg>
        </a>
        <a
          aria-label="Facebook"
          href="https://www.facebook.com/FavForMe/"
          rel="noreferrer noopener"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-facebook fa-w-16 "
            data-icon="facebook"
            data-prefix="fab"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
              fill="currentColor"
            />
          </svg>
        </a>
        <a
          aria-label="Twitter"
          href="https://twitter.com/FavForMe1"
          rel="noreferrer noopener"
          target="_blank"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-twitter fa-w-16 "
            data-icon="twitter"
            data-prefix="fab"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              fill="currentColor"
            />
          </svg>
        </a>
      </div>
      <section
        class="c20"
      >
        <p>
          FavForMe 2021 - All rights reserved
        </p>
        <p
          class="credits"
        >
          Illustrations by 
          <a
            href="https://storyset.com/people"
            rel="noreferrer noopener"
            target="_blank"
          >
            Freepik Storyset
          </a>
        </p>
      </section>
    </section>
  </main>
</DocumentFragment>
`;
